### ISSUE TITLE
Running on windows: Any docs please?

### ISSUE THREAD

**Issue by PreetSangha** [Microsoft/tfs-cli#42]
Is there any documentation at all on this? Of should be be using an alternative utility please?

**Comment by bryanmacfarlane** [comment]
The front readme has the docs and links off to other docs per commands with videos

https://github.com/Microsoft/tfs-cli/blob/master/README.md

If you can be more specific on what you're looking for, we can add it

**Comment by PreetSangha** [comment]
Thank you brian. I apologise for being a complete nexcomer to node so bear
with me. I did go through that. After some hunting I managed to execute the
install with npm  (tfs--cli not tfx-cli). However I couldn't find the
tfs-cli command, except by eventually traversing to the app data folder in
windows and executing node tfs-cli. So my question is basically, do i have
to execute node "....long path ....\tfs-cli each time, or is there
something missing in the docs that tells me how to get a tfx command?

Thanks, preet
